src/extension/_locales/de/messages.json

```diff
@@ -630,6 +633,27 @@
   "user_switch": {
     "message": "Benutzer/in wechseln"
   },
+  "v7_context_menu": {
+    "message": "Installieren Sie die neue AEM Sidekick"
```

Comment:
Use masculine when referring to the sidekick in german.

Reply:
@rofe these should be improved once we get vendor human translation (to replace MT)